src/devices/tpms_eezrv.c

```diff
@@ -14,28 +14,28 @@
 /**
 EezTire E618 TPMS.
 
-Eez RV supported model TPMS10ATC (E618) : https://eezrvproducts.com/shop/ols/products/tpms10atc
+Eez RV supported TPMS sensor model E618 : https://eezrvproducts.com/shop/ols/products/tpms-system-e518-anti-theft-replacement-sensor-1-ea
 
-S.a issue #2384
+S.a issue #2384, #2657
 
 Data layout:
 
-    PRE CC IIIIII PP TT FF 00
+    PRE CC IIIIII PP TT FF FF
 
 - PRE : FFFF
 - C : 8 bit CheckSum
 - I: 24 bit ID
 - P: 8 bit pressure  P * 2.5 = Pressure kPa
 - T: 8 bit temperature   T - 50 = Temperature C
-- F: 8 bit battery (not verified) and deflation pressure (for sure) flags
+- F: 16  bit battery (not verified), deflation pressure (for sure), pressure modifier flags
```

Comment:
`640/2.5 = 256` ;) so please: "pressure MSB"

Reply:
Changes made to add value in flag field (currently on 1 bit) to the pressure as the MSB.